Easy.Common/DiagnosticReport/DiagnosticReport.cs

```diff
@@ -57,7 +57,7 @@ public class DiagnosticReport
             "OS - Name",
             "OS - Type",
             "OS - 64Bit", 
-            "CLR Runtime", 
+            ".NET Version", 
```

Comment:
_CLR Runtime_ is more accurate.

Reply:
I would like to keep that as _CLR Runtime_ to reflect the run-time and not the framework. I may decide to change the value returned by that property.

Reply:
ok, amended commit with only dup line removal